README.md

```diff
@@ -114,17 +114,17 @@ For example, the PyPI cache is shared across builds and thus the package will be
 `envd` can be installed with `pip`, or you can download the binary [release](https://github.com/tensorchord/envd/releases) directly. After the installation, please run `envd bootstrap` to bootstrap.
 
 ```bash
-# install with local pip
-pip install --pre --upgrade envd
+# install with one command
+curl -sSfL https://envd.tensorchord.ai/install.sh | sudo sh
```

Comment:
yes, on macos the `sh` doesn't point to `bash`, fixed it. I also leave only one command in the code block so that users can click the Copy button and paste it to the terminal